components/modal.js

```diff
@@ -137,11 +137,13 @@ const ModalBackground = styled.div`
   /* TODO: add media queries to make width responsive */
   width: 740px;
   max-width: 100%;
-  max-height: 100%;
+  max-height: 70%;
+  overflow: scroll;
```

Comment:
Super small thing: `overflow: scroll;` shows the scroll bar containers for both horizontal and vertical scroll even when there isn't overflow -- this isn't obvious on Apple devices (everything looks better on Apple devices?) but on Windows the scrollbars are kinda chunky. 

Horizontal overflow shouldn't exist so this line could be `overflow-x: auto;`, which would only show vertical scroll when overflow exists

Reply:
OH great call, added in a306420